tests/testthat/test-0diversity.R

```diff
@@ -3,12 +3,13 @@ context("diversity estimates")
 test_that("diversity estimates", {
     skip_if_not(requireNamespace("vegan", quietly = TRUE))
     data("esophagus")
-    esophagus <- estimateDiversity(esophagus)
+    esophagus <- estimateDiversity(esophagus, threshold = 0.473)
```

Comment:
I propose that we use a specific default value for "threshold". This helps to keep it standardized for users who are not sure what to do with it and avoids the need to define it explicitly every time in the function call. A suitable default value is threshold = 0.9

Reply:
Threshold is 0.9